app/models/emis_redis/military_information.rb

```diff
@@ -167,23 +167,16 @@ def va_compensation_type
       # support specs that will be removed or updated
       high_disability = is_va_service_connected
       low_disability = compensable_va_service_connected
-      pension = receives_va_pension
 
       if high_disability
         'highDisability'
-      elsif pension
-        'pension'
       elsif low_disability
         'lowDisability'
       else
         'none'
```

Comment:
Should we remove this `else`? A user could have a VA pension or nothing, but we don't know that for sure.

Reply:
Yeah, that's probably a good call.